app/sliders/config_importer.py

```diff
@@ -50,7 +53,11 @@ def override_values(self, default_config, custom_config):
 
         overwritten_config = default_config.copy()
```

Comment:
@guadiromero are you sure using `.copy()` is the desired behaviour and not `.deepcopy()` ?

Reply:
I don't intend to modify the original dictionary, I think you are right